packages/contracts-periphery/contracts/universal/op-erc721/ERC721Bridge.sol

```diff
@@ -0,0 +1,203 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {
+    CrossDomainMessenger
+} from "@eth-optimism/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol";
+import { Initializable } from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import { Address } from "@openzeppelin/contracts/utils/Address.sol";
+
+/**
+ * @title ERC721Bridge
+ * @notice ERC721Bridge is a base contract for the L1 and L2 ERC721 bridges.
+ */
+abstract contract ERC721Bridge is Initializable {
```

Comment:
Does this need to be `Initializable` now? I see `initialize()` was removed

Reply:
I think its safe to remove `Initializable`

Reply:
Fixed

Reply:
Looks like it's still there?

Reply:
I think you may have been looking at an old commit in this PR. It's been removed in the [latest commit](https://github.com/ethereum-optimism/optimism/pull/3491/files#diff-6cb5db984e0919f9f53eca29b6996b78c8213f875cef4bd516b0a765fcd01a58)